src/app/pages/courses/editCourse/editCourse.component.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  Output,
+  NgZone,
+  EventEmitter,
+  ChangeDetectionStrategy
+} from '@angular/core';
+
+import { Observable } from 'rxjs/Observable';
+import { AppState } from '../../../app.service';
+import { ICourse } from '../iCourse.interface';
+
+@Component({
+  selector: 'app-edit',
+  templateUrl: './editCourse.component.html',
+  styleUrls: ['./editCourse.component.html'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+
+export class EditCourseComponent implements OnInit {
+  @Input() courseData: ICourse;
+  @Output() courseDataOutput = new EventEmitter<any>();
```

Comment:
wiem, że się czepiam, ale warningi tslint są po coś :)
```
WARNING in ./src/app/pages/courses/editCourse/editCourse.component.ts
[23, 3]: The class property 'courseData' must be marked either 'private', 'public', or 'protected'
[24, 3]: The class property 'courseDataOutput' must be marked either 'private', 'public', or 'protected'

 @ ./src/app/pages/courses/editCourse/index.ts 1:0-39
 @ ./src/app/app.module.ts
 @ ./src/app/index.ts
 @ ./src/main.browser.ts
 @ multi (webpack)-dev-server/client?http://localhost:3000 ./src/main.browser.ts
```
jeśli chcesz, żeby tslint traktował brak modyfikator jako public dodaj to `tslint.js`
```
  "rules": {
    "member-access": false,
    ...
```